package.json

```diff
@@ -19,16 +19,27 @@
     "test": "node $NODE_DEBUG_OPTION test/sdp_interop.js",
     "coverage": "jscoverage lib && SDP_TRANSFORM_COV=1 nodeunit --reporter=lcov test",
     "coveralls": "npm run coverage | coveralls",
-    "lint": "./node_modules/.bin/jshint .",
+    "lint": "./node_modules/.bin/eslint .",
     "validate": "npm ls"
   },
   "dependencies": {
     "sdp-transform": "2.3.0"
   },
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^10.1.0",
+    "babel-preset-latest": "^6.24.1",
     "coveralls": "^2.11.1",
+    "eslint": "^6.8.0",
+    "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi#1.0.2",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-flowtype": "^4.7.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
     "jscoverage": "^0.5.5",
-    "jshint": "2.8.0",
+    "lodash.clonedeep": "^4.5.0",
```

Comment:
do we really need all these dependencies or are there some that we can prune?

Reply:
We do need lodash but will see if I can do some pruning.

Reply:
Thanks Jaya. Since you pruned Browserify and migrated over to ESLint, we can probably also remove jshint and watchify?

Reply:
Sure, will do that.

Reply:
Sure, will do that.

Reply:
That's great, thanks Jaya. Since we booted JSHint we should probably remove stale files like `.jshintignore` and `.jshintrc`. Also `npm run lint` fails currently. The error message suggests renaming `.eslintrc.js` to `.eslintrc.cjs`. Could you please check if that fixes the issue?

Reply:
Oh, looks like we get the error with node 13.1.0 and not with 12.13.0.